### ISSUE TITLE
DO anyone could train the same performance as the author provided on voc2007 or voc2012?

### ISSUE THREAD

**Issue by hectorgui** [sanghoon/pva-faster-rcnn#83]

**Comment by hectorgui** [comment]
So, for the 84.9%, the training data set is not voc2007+2012, if the training data set is 2007+2012+coco, I could get the 84.9%? this is my guess, could anyone confirm or correct it?

**Comment by jwnsu** [comment]
That's correct, 84.9% AP is based on additional training data from coco. AP 81% on 2017+2012 is a terrific result, was best result at the time of publication.
